packages/create-sitecore-jss/src/templates/nextjs-sxa/src/pages/api/sitemap.ts

```diff
@@ -58,7 +57,7 @@ const sitemapApi = async (
       const lastSegment = parseUrl[parseUrl.length - 1];
 
       return `<sitemap>
-        <loc>${getPublicUrl()}/${lastSegment}</loc>
+        <loc>${config.publicUrl}/${lastSegment}</loc>
```

Comment:
@yavorsk Can you, please, verify that sitemap api is still works as expected?
Potentially, as @ambrauer mentioned, empty string can't be used here, so it should be tested

Reply:
@matkovskyi can you verify if using empty string breaks your functionality?